Failed integration: `test_jobs_with_no_inventory_database` - Path (/Users/***/.8wzk) doesn't exist.

```
        finally:
            logger.debug(f"cleaning up install folder: {install._install_folder}")
>           ws.workspace.delete(install._install_folder, recursive=True)

tests/integration/test_installation.py:245: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
../../../.local/share/hatch/env/virtual/databricks-labs-ucx/zIWoNST7/integration/lib/python3.10/site-packages/databricks/sdk/service/workspace.py:1629: in delete
    self._api.do('POST', '/api/2.0/workspace/delete', body=body, headers=headers)
../../../.local/share/hatch/env/virtual/databricks-labs-ucx/zIWoNST7/integration/lib/python3.10/site-packages/databricks/sdk/core.py:1110: in do
    return retryable(self._perform)(method,
../../../.local/share/hatch/env/virtual/databricks-labs-ucx/zIWoNST7/integration/lib/python3.10/site-packages/databricks/sdk/retries.py:47: in wrapper
    raise err
../../../.local/share/hatch/env/virtual/databricks-labs-ucx/zIWoNST7/integration/lib/python3.10/site-packages/databricks/sdk/retries.py:29: in wrapper
    return func(*args, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <databricks.sdk.core.ApiClient object at 0x7ff5cbbfc220>, method = 'POST'
path = '/api/2.0/workspace/delete', query = None
headers = {'Accept': 'application/json', 'Content-Type': 'application/json', 'User-Agent': 'ucx/0.5.0 databricks-sdk-py/0.12.0 python/3.10.13 os/linux auth/azure-cli'}
body = {'path': '/Users/***/.8wzk', 'recursive': True}
raw = False, files = None, data = None

    def _perform(self,
                 method: str,
                 path: str,
                 query: dict = None,
                 headers: dict = None,
                 body: dict = None,
                 raw: bool = False,
                 files=None,
                 data=None):
        response = self._session.request(method,
                                         f"{self._cfg.host}{path}",
                                         params=self._fix_query_string(query),
                                         json=body,
                                         headers=headers,
                                         files=files,
                                         data=data,
                                         stream=raw)
        try:
            self._record_request_log(response, raw=raw or data is not None or files is not None)
            if not response.ok: # internally calls response.raise_for_status()
                payload = response.json()
>               raise self._make_nicer_error(response=response, **payload) from None
E               databricks.sdk.core.DatabricksError: Path (/Users/***/.8wzk) doesn't exist.
```

--- a/tests/integration/test_installation.py
+++ b/tests/integration/test_installation.py
@@ -2,7 +2,7 @@ import logging
 from datetime import timedelta
 
 import pytest
-from databricks.sdk.errors import OperationFailed, NotFound
+from databricks.sdk.errors import NotFound, OperationFailed
 from databricks.sdk.retries import retried
 from databricks.sdk.service.catalog import SchemaInfo
 from databricks.sdk.service.iam import PermissionLevel

--- a/tests/integration/test_installation.py
+++ b/tests/integration/test_installation.py
@@ -2,7 +2,7 @@ import logging
 from datetime import timedelta
 
 import pytest
-from databricks.sdk.errors import OperationFailed
+from databricks.sdk.errors import OperationFailed, NotFound
 from databricks.sdk.retries import retried
 from databricks.sdk.service.catalog import SchemaInfo
 from databricks.sdk.service.iam import PermissionLevel
@@ -77,6 +77,7 @@ def test_logs_are_available(ws, sql_backend, env_or_skip, make_random):
     assert len(workflow_run_logs) == 1
 
 
+@retried(on=[NotFound], timeout=timedelta(minutes=15))
 def test_jobs_with_no_inventory_database(
     ws,
     sql_backend,
@@ -247,6 +248,6 @@ def test_jobs_with_no_inventory_database(
         logger.debug(f"cleaning up install folder: {install._install_folder}")
         ws.workspace.delete(install._install_folder, recursive=True)
 
-        for step, job_id in install._deployed_steps.items():
+        for step, job_id in install._state.jobs.items():
             logger.debug(f"cleaning up {step} job_id={job_id}")
             ws.jobs.delete(job_id)

